### PR TITLE
Add tutorials "One gene across biological resources and formats" and "One protein along the UniProt page"

### PR DESCRIPTION
**Pull request by Lisanna**
This pull request will add to the `Foundations of Data Science > Other` topic the two tutorials:

- [One gene across biological resources and formats](https://lisanna.github.io/galaxy-training-material/topics/data-science/tutorials/online-resources-gene/tutorial.html) and 
- [One protein along the UniProt page](https://lisanna.github.io/galaxy-training-material/topics/data-science/tutorials/online-resources-protein/tutorial.html) 

developed for the BioNT workshop, both labeled as drafts and including the introductory slides.

### CODE REVIEW

File: _config.yml [hunk 1]
```diff
@@ -18,7 +18,7 @@ title: "Galaxy Training!"
 email: galaxytrainingnetwork@gmail.com
 description: A collection of tutorials generated and maintained by Galaxy Community Members across the world
 url: "https://training.galaxyproject.org"
-baseurl: "/training-material"
+baseurl: "/galaxy-training-material"
```
Comment (bebatut) [inline _config.yml:21]:
```suggestion
baseurl: "/training-material"
```

**Comment by bebatut** [comment]
@Lisanna is it ready?

**Comment by Lisanna** [comment]
No, sorry. I was waiting to test it on Friday to send you a reviewed version, as my understanding was that you are ok with running the lesson on Friday with a reference to my fork. If instead you would prefer to have it hosted in the main repository, I can send you an updated version now.

**Comment by Lisanna** [comment]
All right, the config fits your settings now. I believe this is ready to be merged. 
Thank you for your help!

**Review by shiltemann — APPROVED** [review]

**Review by bebatut — APPROVED** [review]
Ready to be merged. Thanks a lot @Lisanna for these nice tutorials 😄

**Comment by shiltemann** [comment]
Thanks @Lisanna, great work! :tada:

**Pull request merged**